### DISCUSSION TITLE
Requires explicit mapping/profile for specified Source object

### DISCUSSION THREAD

**Discussion by khoavn** [MapsterMapper/Mapster#711] [Q&A]
Is it possible? The scenario is when using `ProjectToType` with some Source/data table, I need to limit it to some specific Destinations only, when adding new Destination, I expect it must be configurated explicitly.

Thank you!

**Comment by DocSvartz** [comment]
@khoavn  If you want mapping only with manual settings. You can use
` .IgnoreNonMapped(true);`
